src/assets/i18n/en.json

```diff
@@ -254,7 +255,32 @@
         "ai": {
             "title": "AI",
             "links": "<li> <a class=\"link\" href=\"https://www.tutorialspoint.com/artificial_intelligence/index.htm\" target=_blank>AI Tutorial</a></li><li> <a class=\"link\" href=\"http://www.philocomp.net/links/ai.htm\" target=_blank>AI References</a></li>"
+        },
+        "arduino": {
+            "title": "Arduino",
+            "links":"<li> <a class=\"link\" href =\"https://docs.arduino.cc/\" target = _blank>Arduino Documentation</a></li><li> <a class=\"link\" href =\"https://github.com/arduino/\" target = _blank>Arduino Github Resources</a></li>"
+        },
+        "external_API_requests":{
+            "title": "External API Requests",
+            "links": "<li> <a class=\"link\" href =\"https://docs.xano.com/working-with-data/functions/external-api-request  \" target = _blank> What are External API Requests?</a></li><li> <a class=\"link\" href =\"https://github.com/topics/external-api  \" target = _blank>Github Resources </a></li>"
+        },
+        "security_configuration":{
+            "title": " Security Configurations",
+            "links": "<li> <a class=\"link\" href=\"https://www.sciencedirect.com/topics/computer-science/configuration-security \" target=_blank> What are Security Configurations? </a></li><li> <a class=\"link\" href =\" \" target = _blank> Github Repository</a></li>"
```

Comment:
In the `Security Configuration` section, the  GitHub repository link is missing.

Reply:
@josi543 and I couldn’t find a good GitHub repo for it yet so we just have there as a placeholder

Reply:
I thought so! In that case, Ignore my comment

Reply:
I’ll add it to our Trello board under to-do so it doesn’t get forgotten about

Reply:
yes, thank you!